Eevee.Sleep.Bot/Models/InGameAnnouncement/InGame/InGameAnnouncementIndexResponse.cs

```diff
@@ -0,0 +1,52 @@
+using System.Text.Json.Serialization;
+using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Extensions;
+
+namespace Eevee.Sleep.Bot.Models.InGameAnnouncement.InGame;
+
+public record InGameAnnouncementIndexResponse : IInGameAnnouncementResponse<InGameAnnouncementIndexModel> {
+    [JsonPropertyName("id")]
+    public required string Id { get; set; }
+
+    [JsonPropertyName("title")]
+    public required string Title { get; set; }
+
+    [JsonPropertyName("hash")]
+    public required string Hash { get; set; }
+
+    [JsonPropertyName("category")]
+    public required int Category { get; init; }
+
+    [JsonPropertyName("imgUrl")]
+    public required string ImgUrl { get; init; }
+
+    [JsonPropertyName("url")]
+    public required string Url { get; init; }
+
+    [JsonPropertyName("stAt")]
+    public required long StAt { get; init; }
```

Comment:
Should be fine having full name here `StartAt` if I understand what this means correctly